apps/vehicles/test/repo_test.exs

```diff
@@ -34,9 +34,13 @@ defmodule Vehicles.RepoTest do
 
   describe "route/1" do
     test "given a route ID, finds vehicle statuses for that route", %{name: name} do
-      expected = Enum.filter(@vehicles, &(&1.route_id == "86"))
-      refute Enum.empty?(expected)
-      assert Repo.route("86", name: name) == expected
+      expected_ids = Enum.filter(@vehicles, &(&1.route_id == "86")) |> Enum.map(& &1.id)
+
+      refute Enum.empty?(expected_ids)
+
+      for result <- Repo.route("86", name: name) do
+        assert Enum.member?(expected_ids, result.id)
+      end
```

Comment:
suggestion: this test has the same issue, but the other way around. values that are in `expected_ids` but not in `Repo.route("86", name: name)` are not validated.